File: spring/demo/src/main/java/com/example/demo/LoadSimulator.java
package com.example.demo;

import java.net.URI;
import java.net.http.*;
import java.util.stream.IntStream;

public class LoadSimulator {
    public static void main(String[] args) throws Exception {
        HttpClient client = HttpClient.newHttpClient();

        IntStream.range(0, 5000).forEach(i -> {
            new Thread(() -> {
                try {
                    HttpRequest request = HttpRequest.newBuilder()
                            .uri(URI.create("http://localhost:8080/hello"))
                            .build();
                    HttpResponse<String> response =
                            client.send(request, HttpResponse.BodyHandlers.ofString());
                    System.out.println(response.body());
                } catch (Exception e) {
                    e.printStackTrace();
                }
            }).start();
        });
    }
}

File: dotnet/Lab01/LoadSimulator/LoadSimulator.cs
using System;
using System.Threading.Tasks;
using System.Net.Http;
class LoadSimulator
{
    static async Task Main(string[] args)
    {
        using var client = new HttpClient();
        var tasks = new Task[50];
        for (int i = 0; i < 50; i++)
        {
            tasks[i] = Task.Run(async () =>
            {
                try
                {
                    var response = await client.GetStringAsync("http://localhost:5280/hello");
                    Console.WriteLine(response);
                }
                catch (Exception ex)
                {
                    Console.WriteLine($"Error: {ex.Message}");
                }
            });
        }
        await Task.WhenAll(tasks);
    }
}